src/Error/Debugger.php

```diff
@@ -626,6 +626,23 @@ public static function exportVar($var, int $maxDepth = 3): string
         return static::getInstance()->getExportFormatter()->dump($node);
     }
 
+    /**
+     * Convert the variable to the internal node tree.
+     *
+     * The node tree can be manipulated and serialized more easily
+     * than many object graphs can.
+     *
+     * @param mixed $var Variable to convert.
+     * @param int $maxDepth The depth to generate nodes to. Defaults to 3.
+     * @return NodeInterface The root node of the tree.
+     */
+    public static function exportNodes($var, int $maxDepth = 3): NodeInterface
```

Comment:
This shouldn't be a big debate, but this name feels a little odd for what it does. Why don't we just expose `export` and have users pass in a context?

Reply:
I agree the name is a bit clunky. Would `exportToNodes` be better? I was concerned with requiring callers to have to construct the context and be exposed to more of the conversion process guts.

Reply:
To keep it clear between `exportVar()` and this, I would say `exportVarAsNodes()`. exportNodes sounds like it takes nodes as input and exports debug string like exportVar does.